src/Commands/MakeModelCommand.php

```diff
@@ -72,6 +65,30 @@ public function handle(): void
                 )
             );
 
+        $this->resolveForMigration(name: $name);
+
+        $this->resolveForFactory(name: $name);
+
         $this->info(string: 'Successfully generate model file');
     }
+
+    protected function resolveForFactory(string $name): void
+    {
+        if ($this->option(key: 'factory')) {
```

Comment:
Put commons either option or argument in trait

Reply:
it not a common argument i think, factory just related with models

Reply:
Ok can ignore